CorsixTH/Lua/humanoid_actions/idle.lua

```diff
@@ -33,6 +33,11 @@ end
 --!param direction (string) Direction of facing.
 --!return (action) Self, for daisy-chaining.
 function IdleAction:setDirection(direction)
+  assert(direction == nil or
+      direction == "north" or direction == "south" or
+      direction == "east" or direction == "west",
+      "direction must be north, south, east, or west")
```

Comment:
Misses 'nil' as possible value

I am not so sure about the usefulness of the messages, as they seem to duplicate the code, but if you like them, why not.